### PR TITLE
fix(crane): cors

### LINKED ISSUE #516: v0.7.0 Crane集群管理里添加其他集群时发生错误
**Describe the bug**
在两个集群里部署了Crane服务，在集群A里添加集群B时 发生错误。
**Reproduce steps**
两个集群Crane服务从0.6.0升级到0.7.0
**Expected behavior**

**Screenshots**
集群A 103.153 服务正常：
a

集群B 103.168 服务正常：
b

集群A 集群B 网络正常：
image

相互添加报错：
image
image

**Environment (please complete the following information):**
 - K8S Version: [A: v1.19.3 B: v1.23.0]
 - Crane Version: [0.7.0]
 - Browser [chrome]

### PR DESCRIPTION
**Pull request by lbbniu**
…nd the crane cors configuration, resulting in the failure of the cors configuration

#### What type of PR is this?
Solve the conflict between the nginx cors configuration and the crane cors configuration, resulting in the failure of the cors configuration

#### Which issue(s) this PR fixes:

Fixes #516 #538

### CODE REVIEW

**Comment by qmhu** [comment]
@PatrickLai7528  PTAL

File: deploy/craned/deployment.yaml [hunk 1]
```diff
@@ -215,7 +215,6 @@ data:
         add_header Cache-Control "must-revalidate";
         add_header Cache-Control "max-age=300";
         add_header ETag "1.90.0-rc.0";
-        add_header Access-Control-Allow-Origin *;
```
Comment (PatrickLai7528) [inline deploy/craned/deployment.yaml:218]: /grafana required this header.
please add 'add_header Access-Control-Allow-Origin *;' in line 230 @lbbniu

**Comment by lbbniu** [comment]
image

**Review by PatrickLai7528 — APPROVED** [review]

File: deploy/craned/deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbbniu) [reply deploy/craned/deployment.yaml:218]: 添加了

[the remark this answers is not in the document]
Comment (lbbniu) [reply deploy/craned/deployment.yaml:218]: @PatrickLai7528

**Pull request closed without merging**